pynamodb/models.py

```diff
@@ -975,3 +1022,28 @@ def _serialize_keys(cls, hash_key, range_key=None):
         if range_key is not None:
             range_key = cls._range_key_attribute().serialize(range_key)
         return hash_key, range_key
+
+
+class _ModelFuture:
+    """
+    A placeholder object for a model that does not exist yet
+
+    For example: when performing a TransactGet request, this is a stand-in for a model that will be returned
+    when the operation is complete
+    """
+    def __init__(self, model_cls):
+        self._model_cls = model_cls
+        self._model = None
+        self._resolved = False
+
+    def update_with_raw_data(self, data):
+        if data:
```

Comment:
Should this be `if data is not None` or is it intentionally trying to capture some falsy case other than `None`? If so, it's not apparent to me what it would be.

Reply:
it can just be `if data is not None`